src/NServiceBus.Gateway.Tests/APIApprovals.cs

```diff
@@ -17,7 +17,7 @@ public void Approve()
 #else
             var targetFramework = "netstandard";
 #endif
-            var publicApi = ApiGenerator.GeneratePublicApi(typeof(GatewaySettings).Assembly);
+            var publicApi = ApiGenerator.GeneratePublicApi(typeof(GatewaySettings).Assembly, excludeAttributes: new[] { "System.Runtime.Versioning.TargetFrameworkAttribute", "System.Reflection.AssemblyMetadataAttribute" });
```

Comment:
I think I would like to keep the TargetFrameworkAttribute in the files since we are splitting the approval files by target framework.

Reply:
Given it's a release branch, I'd say we keep it as-is.

Reply:
Keeping it as-is, would be to not add the exclusion. This PR is adding it.

Reply:
Ah, right. Fixed.